.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: false
+dist: trusty
+language: erlang
+otp_release:
+  - 20.0.1
+  - 19.3.6.1
+  - 18.3
+addons:
+  apt:
+    sources:
+      - debian-sid
+    packages:
+      ## GNU Make 4.  GNU Make 3.81 as [in Ubuntu 14.04
+      ## "trusty"](https://packages.ubuntu.com/trusty/make) is not
+      ## sufficient, because of error on `triq` test dep when running
+      ## `make tests`.  Also erlang.mk reports the following warning:
+      ## "erlang.mk:24: Please upgrade to GNU Make 4 or later:
+      ## https://erlang.mk/guide/installation.html".
+      - make
+before_install:
+  - make -v
```

Comment:
`make -v` prints version information about GNU Make. I guess you meant `make V=1`. However probably this line can be dropped altogether, as `make edoc` and `make check` will trigger compilation.

Reply:
I meant to print the make version, as I noticed Make 3.81 was not working and I added the addons section for installing Make 4.

Reply:
Yes, this line could be dropped, even if I suggest keeping it for tracking it in case of future failed builds.

Reply:
Ok, we can leave it then, in the form of `make --version` (so that it is not confused with `make --verbose`, even despite of the fact that there is no such option).